pkg/operator/starter.go

```diff
@@ -172,14 +171,7 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 		controllerConfig.EventRecorder,
 	).WithAPIServiceController(
 		"openshift-apiserver",
-		apiservice.NewAPIServicesToManage(
```

Comment:
we should also remove https://github.com/openshift/cluster-openshift-apiserver-operator/blob/d54f7de5ad41056faa9d30e6e2e306ec5b850ecc/pkg/operator/apiservice/apiservice_openshiftapiserver_sync.go

Reply:
I thought I did that, should be gone now :+1: